Eastern-Research-Group/expert-query
# Feature/220 UI column order

## Related Issues:
* [EQ-220](https://jira.epa.gov/browse/EQ-220)

## Main Changes:
* Added a list of columns to the query requests sent to the server, in the order displayed on the UI.
  * Displayed fields are sent first, with any "source" fields sent in front of the corresponding field, then any leftover columns are added to the end.
  * Updated the copy boxes to include the columns for the **API Query** and **cURL** boxes. This significantly increases their size, but it has the benefit of showing "advanced" users that columns can be added/removed/reordered.
  * Stopped `parameterStateIrCategory` and `useStateIrCategory` from using the "StateIRCategoryCode" domain values. The initial templates claimed these domains to be the same, but the actual values and column types don't match.

## Steps To Test:
1. Navigate to http://localhost:3000/attains/assessments#assessmentUnitStatus=A&parameterStateIrCategory=24&useStateIrCategory=24.
2. Execute the query, using either the download button or an "advanced" query, and confirm that no data type errors are raised.
3. In the result set, confirm that the columns are listed in the order that they appear in the interface. Columns that are not included in the interface should be listed at the end.
